HW-1/task2.py

```diff
@@ -13,10 +13,10 @@
     print("Данный треугольник существует.")
 
     if a == b and b == c and c == a:
```

Comment:
Так будет чуть проще
```suggestion
    if a == b == c:
```